### ISSUE TITLE
StopPointType

### ISSUE THREAD

**Issue by mathewtrivett** [mathewtrivett/tfl-graph#27]
## Acceptance criteria

- [x] It should have an `accessibilitySummary: String`
- [x] It should have an `additionalProperties: [AdditionalPropertiesType]` #32 
- [x] It should have a `children: [PlaceType]` #33
- [x] It should have a `childrenUrls: [String]`
- [x] It should have a `commonName: String`
- [x] It should have a `distance: Float`
- [x] It should have a `fullName: String`
- [x] It should have a `hubNaptanCode: String`
- [x] It should have an `icsCode: String`
- [x] It should have an `id: String`
- [x] It should have an `indicator: String`
- [x] It should have a `lat: Float`
- [x] It should have a `lineGroup: [LineGroupType]` #34 
- [x] It should have a `lineModeGroups: [LineModeGroupType]` #35
- [x] It should have a `lines: [IdentifierType]` #36 
- [x] It should have a `lon: Float`
- [x] It should have a `modes: [String]`
- [x] It should have a `naptanId: String`
- [x] It should have a `naptanMode: String`
- [x] It should have a `placeType: String`
- [x] It should have a `platformName: String`
- [x] It should have a `smsCode: String`
- [x] It should have a `stationNaptan: String`
- [x] It should have a `status: Boolean`
- [x] It should have a `stopLetter: String`
- [x] It should have a `stopType: String`
- [x] It should have an `url: String`

## Schema

```graphql
type TflApiPresentationEntitiesStopPoint {
  accessibilitySummary: String

  """
  A bag of additional key/value pairs with extra information about this place.
  """
  additionalProperties: [TflApiPresentationEntitiesAdditionalProperties]
  children: [TflApiPresentationEntitiesPlace]
  childrenUrls: [String]

  """A human readable name."""
  commonName: String

  """
  The distance of the place from its search point, if this is the result
              of a geographical search, otherwise zero.
  """
  distance: Float
  fullName: String
  hubNaptanCode: String
  icsCode: String

  """A unique identifier."""
  id: String

  """
  The indicator of the stop point e.g. "Stop K"
  """
  indicator: String

  """WGS84 latitude of the location."""
  lat: Float
  lineGroup: [TflApiPresentationEntitiesLineGroup]
  lineModeGroups: [TflApiPresentationEntitiesLineModeGroup]
  lines: [TflApiPresentationEntitiesIdentifier]

  """WGS84 longitude of the location."""
  lon: Float
  modes: [String]
  naptanId: String
  naptanMode: String

  """The type of Place. See /Place/Meta/placeTypes for possible values."""
  placeType: String
  platformName: String
  smsCode: String
  stationNaptan: String
  status: Boolean

  """
  The stop letter, if it could be cleansed from the Indicator e.g. "K"
  """
  stopLetter: String
  stopType: String

  """The unique location of this resource."""
  url: String
}

```
